scania-digital-design-system/sdds
# feat(radio button): web component

**Describe pull-request**  
Web component version of the radio button component. Also renamed native story to Native.

**Solving issue**  
Fixes: [AB#3125](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/3125)

**How to test**  
    1. Go to story book link below
    2. Check in Component -> Radio Button-> Web Component
    3. Check that styling and functionality is correct
   
**Suggested test steps**

**Screenshots**  

**Additional context**
